### ISSUE TITLE
Detection of non-existent records

### ISSUE THREAD

**Issue by primem0ver** [TES5Edit/TES5Edit#1059]
I encountered this problem when using DynDoLOD.  I currently get 4 errors that report REFR record over-writing other kinds of records.  The first two existed before and don't involve a merged mod.  I haven't verified whether these are real or not.  However, after merging some mods recently because of having some problems with the first set of mods not being entirely compatible, I now get another two reported errors like this.  I have spent a good 6 hours looking for what could cause this and have found nothing.  I also checked in xEdit and sure enough, it is reporting two MGEF have altered records (red).  When I check these, there are blank records adjacent to them in the merge mod's column.  Yet these records do not exist in the merge.  As mentioned, I did a pretty thorough search for them:

1. I searched for the record ids in xEdit and only the ones in the original (master) file appear.  No records show as being found in the merged mod.
2. I checked the merge record log (lists ALL copied records and their new ids). 
3. I checked the binary file for the record id (yes... I did in little endian fashion)
4. I realized I didn't need to do this afterward but I even checked the compressed records for any evidence of the form ID of the reportedly replaced record and found none (I have a utility that can decompress the records).  

Not sure what is causing this false detection but... it doesn't appear to be an actual record.

**Comment by ElminsterAU** [comment]
a) this is for xEdit, not DynDoLOD.
b) It's is exceedingly unlikely that whatever you problem is a bug in xEdit, and you are not making a usable bugreport in any form or shape here.
c) support question belong on the xEdit Discord, not in a GitHub issue.

**Comment by primem0ver** [comment]
Perhaps you did not read above?  "I also checked in xEdit and sure enough, it is reporting two MGEF have altered records (red).  When I check these, there are blank records adjacent to them in the merge mod's column."  This is why I believe this is an issue with xEdit.  

As you probably know, several tools use the xEdit codebase which is why I felt it appropriate to report here as yesterday I misunderstood what discord wanted my phone number for in order to post on Discord (which I did attempt before coming here).  I will go ahead and report it there.

Basically xEdit is detecting the same problem. There are TWO columns side by side for the said record in xEdit when there should only be one.  One column for the original file (with data), the other for the merge (blank).  This means xEdit is detecting the "Record" and I am guessing the reason it is blank is because it's record signature doesn't match so there is nothing to compare (or perhaps as I suggest it is because there is no real record and hence no data to actually read.  It doesn't even show a record signature to compare).   If you want I can upload a screenshot later to show you what I mean.  I can't right now because a utility is running on the files in my load order.
